flexeval/core/language_model/openai_batch_api.py

```diff
@@ -136,13 +141,14 @@ def _execute_batch_requests(
         custom_id_2_message: dict[str, list[dict[str, str]]] = {
             str(uuid.uuid4()): messages for messages in messages_list
         }
-        custom_id_2_response: dict[str, str | None] = {custom_id: None for custom_id in custom_id_2_message}
-        exec_cnt = 0
+        custom_id_2_response: dict[str, str] = {custom_id: "" for custom_id in custom_id_2_message}
+        exec_cnt = 1
```

Comment:
How about adding a commend to emphasize the empty string?

```suggestion
        # The response will be an empty string if the API produces an error.
        custom_id_2_response: dict[str, str] = {custom_id: "" for custom_id in custom_id_2_message}
        exec_cnt = 1
```